spec/use_cases/administrator/create_support_ticket_spec.rb

```diff
@@ -0,0 +1,25 @@
+describe UseCases::Administrator::CreateSupportTicket do
+  subject { described_class.new(tickets_gateway: tickets_gateway_spy) }
+
+  let(:tickets_gateway_spy) { spy(:create) }
+
+  before do
+    subject.execute(
+      requester: {
+        email: 'help@support.com',
+        name: 'Helpy McHelpface',
+        organisation: 'Parks & Rec'
```

Comment:
We don't use organisation in the request?

Reply:
We don't. My intent is to add to the body of the support request, but I skipped it for now as I'm worried about time. 

Happy to work on adding it in now if you think it'd be a big miss - haven't seen a support ticket yet where it's been essential to providing a good response.